en/snap/interactive-demo.md

```diff
@@ -1,46 +1,47 @@
-# Snap Interactive Integration Demo
+# Interactive Demonstration of Snap Integration
 <hr>
-Snap integration will be explained simply using interactive demo. Which you can interactively try the Snap integration demo below. Inspect the source code and see the live output.
 
-### Preparation
+Step 1 and Step 2 in [Snap Integration Overview](/en/snap/overview.md) is explained here using interactive demonstration. You can try the *Snap* integration demo given below. You can inspect the source code and see the real-time output.
+
+## Preparations
 <br>
 <div class="my-card">
 
 #### [Sign Up for Midtrans Account](/en/midtrans-account/overview.md)
-Sign up for an Account to get your Sandbox API keys ready to test integration.
+Sign up for a Midtrans Merchant Administration Portal (MAP) account, to get your API Keys for *Sandbox* environment and to test integration.
 </div>
 
 <div class="my-card">
 
-#### [Retrieve API Keys](/en/midtrans-account/overview.md#retrieving-api-access-keys)
-Retrieve Sandbox mode API keys that will be used for this guide.
+#### [Retrieving API Keys](/en/midtrans-account/overview.md#retrieving-api-access-keys)
+Retrieve API Keys for *Sandbox* environment that will be used for this guide.
 </div>
 
-### Technology Stack
-
-The sample integration's:
-- **Backend** is **NodeJS** (hosted on CodeSandbox).
-- **Frontend** is plain **HTML & Javascript**.
-
-### Usage Explanation
+## Requirements
```

Comment:
> Requirements

Is not quite the correct term. This section is not explaining what is required, it is simply explaining what "technology stack" being used for the demo.

It is a demo, nothing is required. It is explaining what the demo use. It is like when you see an Painting Demonstration, and the Painter tell you what he used to do the Painting, he explains that he use "water-color paint, canvas, brush, etc.". He does not require you.

Reply:
Nice, other than the feedback above, everything else looks good for this page 👍